apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -358,7 +374,9 @@ Func CameraPipe::build() {
         .unroll(c)
         .parallel(yo);
 
-    demosaiced_scheduler(processed);
+    demosaiced->intermed_compute_at.set(LoopLevel(processed, yi));
```

Comment:
intermed_compute_at.set({processed, yi}); 

feels a little more natural to me. Or maybe ScheduleParam<Foo>::set should be variadic and take anything you can construct a Foo from. Then we'd get:

   intermed_compute_at.set(processed, yi);